pf/tests/util.go

```diff
@@ -86,24 +110,29 @@ func ensureProviderValid(prov *providerbuilder.Provider) {
 	}
 }
 
-func bridgedProvider(prov *providerbuilder.Provider) info.Provider {
+func bridgedProvider(prov *providerbuilder.Provider, opts ...ProviderOption) info.Provider {
```

Comment:
I don't really see why we need the options here - the caller receives the full info.Provider, so is free to modify it as they see fit - the options here will just expand to the full interface as we add tests.

Reply:
🤦🏼‍♂️